mm2src/mm2_test_helpers/src/structs.rs

```diff
@@ -904,20 +904,28 @@ pub struct CoinAddressInfo<Balance> {
 
 pub type TokenBalances = HashMap<String, CoinBalance>;
 
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct EnableEthWithTokensResponse {
+    pub current_block: u64,
+    pub eth_addresses_infos: Option<HashMap<String, CoinAddressInfo<CoinBalance>>>,
+    pub erc20_addresses_infos: Option<HashMap<String, CoinAddressInfo<TokenBalances>>>,
```

Comment:
`erc20_addresses_info`

Reply:
This comment https://github.com/KomodoPlatform/atomicDEX-API/pull/1762#discussion_r1178192537 applies here too.